Modules/Devices/src/Devices.Infrastructure/PushNotifications/Connectors/Dummy/DummyConnector.cs

```diff
@@ -1,43 +1,43 @@
 using Backbone.BuildingBlocks.Application.PushNotifications;
-using Backbone.DevelopmentKit.Identity.ValueObjects;
-using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
 using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
-using Backbone.Modules.Devices.Infrastructure.PushNotifications.Connectors.Apns;
-using Backbone.Modules.Devices.Infrastructure.PushNotifications.NotificationTexts;
 using Backbone.Modules.Devices.Infrastructure.PushNotifications.Responses;
 using Microsoft.Extensions.Logging;
 
 namespace Backbone.Modules.Devices.Infrastructure.PushNotifications.Connectors.Dummy;
 
 public class DummyConnector : IPnsConnector
 {
-    private readonly IIdentitiesRepository _identitiesRepository;
-    private readonly IPushNotificationTextProvider _notificationTextProvider;
-    private readonly ILogger<ApplePushNotificationServiceConnector> _logger;
+    private readonly ILogger<DummyConnector> _logger;
 
-    public DummyConnector(IIdentitiesRepository identitiesRepository, IPushNotificationTextProvider notificationTextProvider, ILogger<ApplePushNotificationServiceConnector> logger)
+    public DummyConnector(ILogger<DummyConnector> logger)
     {
-        _identitiesRepository = identitiesRepository;
-        _notificationTextProvider = notificationTextProvider;
         _logger = logger;
     }
 
-    public async Task<SendResults> Send(IEnumerable<PnsRegistration> registrations, IdentityAddress recipient, IPushNotification notification)
+    public Task<SendResults> Send(IEnumerable<PnsRegistration> registrations, IPushNotification notification)
     {
-        var sendResults = new SendResults();
-
-        var identity = await _identitiesRepository.FindByAddress(recipient, CancellationToken.None) ?? throw new Exception("Identity not found.");
+        _logger.Sending(notification.GetEventName());
 
-        foreach (var device in identity.Devices)
+        var sendResults = new SendResults();
+        foreach (var registration in registrations)
         {
-            var (title, body) = await _notificationTextProvider.GetNotificationTextForDeviceId(notification.GetType(), device.Id);
-            _logger.LogInformation("Sending push notification to device with id '{deviceId}' of identity with address '{recipient}': {title}, {body}.", recipient, device.Id, title, body);
+            sendResults.AddSuccess(registration.DeviceId);
         }
 
-        return sendResults;
+        return Task.FromResult(sendResults);
     }
```

Comment:
So just to clarify in order to see if I understand the code correctly - since this is a dummy we are not really sending anything, but instead are adding a success to results for each of the registrations. Correct?

Reply:
Correct